policy/policy.go

```diff
@@ -30,47 +31,71 @@ type branchVals struct {
 	UpgradeFromVer string                // Versions to test package upgrades from
 	PCPrivate      bool                  // indicates whether package cloud repo is private
 	Branch         map[string]branchVals `copier:"-"`
+	ReviewCount    string
+	Convos         bool
+	Tests          []string
+	SourceBranch   string
 }
 
 // Policies models the config file structure. The config file may contain one or more repos.
 type Policies struct {
-	Description string
-	PCRepo      string
-	DHRepo      string
-	CSRepo      string
-	PackageName string
-	Reviewers   []string
-	ExposePorts string
-	Binary      string
-	Protected   []string
-	Goversion   string
-	Default     string              // The default git branch(master/main/anything else)
-	Repos       map[string]Policies // map of reponames to branchPolicies
-	Ports       map[string][]string
-	Branches    branchVals
+	Description     string
+	PCRepo          string
+	DHRepo          string
+	CSRepo          string
+	PackageName     string
+	Reviewers       []string
+	ExposePorts     string
+	Binary          string
+	Protected       []string
+	Goversion       string
+	Default         string              // The default git branch(master/main/anything else)
+	Repos           map[string]Policies // map of reponames to branchPolicies
+	RepoPolicy      map[string]RepoPolicy
+	Ports           map[string][]string
+	Branches        branchVals
+	ReleaseBranches []string
 }
 
 // RepoPolicy extracts information from the Policies type for one repo. If you add fields here, the Policies type might have to be updated, and vice versa.
 type RepoPolicy struct {
-	Name        string
-	Description string
-	Protected   []string
-	Default     string
-	PCRepo      string
-	DHRepo      string
-	CSRepo      string
-	Binary      string
-	PackageName string
-	Reviewers   []string
-	ExposePorts string
-	Files       map[string][]string
-	Ports       map[string][]string
-	gitRepo     *git.GitRepo
-	Branch      string
-	prBranch    string
-	Branchvals  branchVals
-	prefix      string
-	Timestamp   string
+	Name            string
+	Description     string
+	Protected       []string
+	Default         string
+	PCRepo          string
+	DHRepo          string
+	CSRepo          string
+	Binary          string
+	PackageName     string
+	Reviewers       []string
+	ExposePorts     string
+	Files           map[string][]string
+	Ports           map[string][]string
+	gitRepo         *git.GitRepo
+	Branch          string
+	ReleaseBranches map[string]branchVals
+	prBranch        string
+	Branchvals      branchVals
+	prefix          string
+	Timestamp       string
+}
+
+func (p *Policies) GetAllRepos(repos []string) (Policies, error) {
```

Comment:
The list of repos should be extracted from the Policies object of which this is a method, not passed as an argument.